src/controllers/userController.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const {
+  generateUniqueID,
+  findUser,
+  getAllUsers,
+  findIndexOfUser,
+  deleteUserByIndex,
+  addNewUser,
+} = require('../services/userService.js');
+const { checkId, checkIfItemFound } = require('../helpers.js');
+
+const getUsers = (req, res) => {
+  if (!getAllUsers()) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.status(200).send(getAllUsers());
+};
```

Comment:
why do you need to run getAllUsers twice?

`const users = getAllUsers();`

and operate the `users` variable